Is the `storybook-readme` package missing from `package.json`? When I run storybook I get errors:

    Module not found: Error: Can't resolve 'storybook-readme' in '/Users/josh/src/tc/tc-assets/Components/Spinner'

--- a/Components/Spinner/README.md
+++ b/Components/Spinner/README.md
@@ -1,0 +1,10 @@
+# Spinner
+
+The Spinner indicates that something is loading, and the user has to wait.
+
+## Properties
+
+ propName | propType | defaultValue | isRequired
+----------|----------|--------------|:----------:
+ enabled  | Boolean  | true         |
+ title    | String   | "Loading"    |

--- a/Components/Spinner/index.jsx
+++ b/Components/Spinner/index.jsx
@@ -1,0 +1,22 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styles from "./styles.css";
+
+const Spinner = ({ enabled, title }) => {
+  if (enabled) {
+    return <span className={styles.spinner} title={title} />;
+  }
+  return null;
+};
+
+Spinner.defaultProps = {
+  enabled: true,
+  title: "Loading",
+};
+
+Spinner.propTypes = {
+  enabled: PropTypes.bool,
+  title: PropTypes.string,
+};
+
+export default Spinner;

--- a/Components/Spinner/spec.jsx
+++ b/Components/Spinner/spec.jsx
@@ -1,0 +1,34 @@
+import React from "react";
+import { shallow } from "enzyme";
+import Spinner from "./";
+
+describe("<Spinner />", () => {
+  describe("default props", () => {
+    test("is enabled by default with the \"Loading\" title", () => {
+      const wrapper = shallow(<Spinner />);
+      expect(wrapper.find("span")).toHaveLength(1);
+      expect(wrapper.find("span").prop("title")).toEqual("Loading");
+    });
+  });
+
+  describe("when enabled", () => {
+    test("should render a span", () => {
+      const wrapper = shallow(<Spinner enabled />);
+      expect(wrapper.find("span")).toHaveLength(1);
+    });
+  });
+
+  describe("when disabled", () => {
+    test("should not render a span", () => {
+      const wrapper = shallow(<Spinner enabled={false} />);
+      expect(wrapper.find("span")).toHaveLength(0);
+    });
+  });
+
+  describe("with a title", () => {
+    test("should render a span with the title", () => {
+      const wrapper = shallow(<Spinner enabled title="boo" />);
+      expect(wrapper.find("span").prop("title")).toEqual("boo");
+    });
+  });
+});

--- a/Components/Spinner/stories.jsx
+++ b/Components/Spinner/stories.jsx
@@ -1,0 +1,30 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { doc } from 'storybook-readme';
+import README from './README.md';
+import Spinner from "./index";
+
+storiesOf('Spinner', module).add('Documentation', doc(README));
+storiesOf("Spinner", module).addWithChapters('Display',  {
+  chapters: [
+    {
+      sections: [
+        {
+          title: "Standard",
+          subtitle: "Default set to 'loading'",
+          info: "Indicates a neutral informative change or action.",
+          sectionFn: () => (
+            <Spinner title="Hello">Button text</Spinner>
+          )
+        }, {
+          title: "Enabled",
+          info: "If set to false, the spinner is no longer shown.",
+          subtitle: "Default set to true",
+          sectionFn: () => (
+            <Spinner enabled={false}>Button text</Spinner>
+          )
+        }
+      ]
+    }
+  ]
+});

--- a/Components/Spinner/stories.jsx
+++ b/Components/Spinner/stories.jsx
@@ -10,9 +10,8 @@ storiesOf("Spinner", module).addWithChapters('Display',  {
     {
       sections: [
         {
-          title: "Standard",
-          subtitle: "Default set to 'loading'",
-          info: "Indicates a neutral informative change or action.",
+          title: "Default",
+          info: "The default spinner is used to indicate to the user that something is happening.",
           sectionFn: () => (
             <Spinner title="Hello">Button text</Spinner>
           )

--- a/Components/Spinner/stories.jsx
+++ b/Components/Spinner/stories.jsx
@@ -4,7 +4,7 @@ import { doc } from 'storybook-readme';
 import README from './README.md';
 import Spinner from "./index";
 
-storiesOf('Spinner', module).add('Documentation', doc(README));
+storiesOf('Spinner', module).add('Overview', doc(README));
 storiesOf("Spinner", module).addWithChapters('Display',  {
   chapters: [
     {

--- a/Components/Spinner/styles.css
+++ b/Components/Spinner/styles.css
@@ -1,0 +1,17 @@
+@keyframes :local(spinnerKeyframes) {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+:local .spinner {
+  animation: spinnerKeyframes 0.8s infinite linear;
+  display: block;
+  width: 12px;
+  height: 12px;
+  margin: 0 auto;
+  border: 3px solid #d9f3fc;
+  border-top: 3px solid #33bef2;
+  border-radius: 50%;
+  transform-origin: 50% 50%;
+  transition: opacity 500ms;
+}

